addon/tern/tern.js

```diff
@@ -589,7 +589,11 @@
       cm.off("cursorActivity", clear);
       fadeOut(tip);
     }
-    setTimeout(clear, 1700);
+    function clearWithTimeout() {
+      cm.getWrapperElement().removeEventListener("mousemove", clearWithTimeout);
```

Comment:
Wouldn't tooltip appear because of mouse move in the first place?

Movement might continue for a few pixels after the tooltip shown, triggering the event immediately anyway.